src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -781,9 +780,15 @@ async Task StrategyExecuteCmdletAsync(IAsyncCmdlet asyncCmdlet)
                     this.EnableVtpm = this.EnableVtpm ?? true;
                     this.EnableSecureBoot = this.EnableSecureBoot ?? true;
                 }
+                // Default in TL Image to ease later migration. 
```

Comment:
i see this is the only part updated for New-AzVM,  
does this code path apply to both Simple and Default parameter set ?

Reply:
The ask is for only this parameter set, I have not tested the other.